README.md

```diff
@@ -2,56 +2,13 @@
 
 ## Background
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+This project builds an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+The database is in PSQL, and will be interacted with using [node-postgres](https://node-postgres.com/).
 
 ## Kanban
 
-### Link to your Trello Board here: https://trello.com/b/7yiHe1nI
+### Link to Trello Board: https://trello.com/b/oGLgYiZo/be-nc-news
```

Comment:
This will be the readme for someone who wants to set up your project and therefore doesnt really need to have a link to your trello board. Just instructions on how to set the project up for someone who wants to try it out